lib/src/message_grouper_state.dart

```diff
@@ -64,7 +64,7 @@ class _LengthReader {
   /// If [_done] is `true`, the decoded value of the length bytes received so
   /// far (if any).  If [_done] is `false`, the decoded length that was most
   /// recently received.
```

Comment:
[out of scope for this review] Can you fix this comment while you are here? This does not appear true, if `_done` is `false` then this will have no value at all and reading it will be an error.

Reply:
Done